react-native/training/src/app/screens/Dummy/components/List/components/RenderItem/components/Picture/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import { View, Image } from 'react-native';
+import { Icon } from 'react-native-elements';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class Picture extends Component {
+  getImage = url =>
```

Comment:
This component can be a functional component. Move this condition to the render method.